pkg/reconciler/apis/apibinding/apibinding_controller.go

```diff
@@ -176,7 +212,7 @@ func (c *controller) enqueueAPIBinding(obj interface{}) {
 		return
 	}
 
-	klog.Infof("Queueing APIBinding %q", key)
+	klog.V(4).Infof("Queueing APIBinding %q", key)
```

Comment:
this might be worth a Slack thread and an update to the contributor guide to declare some verbosity level for different kinds of controller logging. We run in `V(2)` now by default. So `V(4)` seems to be fine, but maybe also `V(2)` might make sense for the main resource queuing in a controller.

Reply:
👍. If V(2) is the default, I'm wondering if we should instead move those 2s to just klog.Info and go with the unset (0/1) default?

Do you think the kcp admin would always want to see "Queueing xyz" for a standard deployment? Or only when running in a more verbose mode for debugging?